### DISCUSSION TITLE
Community call April

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#3823] [Community Call]
Here we all get together monthly on [EddieHub Discord](http://discord.eddiehub.org) to chat about...

- ✅ What you have been learning
- ✅ Any challenges you have had
- ✅ Get more eyes on your content
- ✅ Improvements and suggestions to EddieHub
and more!

Below are some threads to get you started, you can start a new thread to ask your questions etc...

**Comment by eddiejaoude** [comment]
Give yourself an introduction and share your links...

**Comment by eddiejaoude** [comment]
Suggestions and improvements for EddieHub...

**Comment by eddiejaoude** [comment]
Share 1 link for a social post or content you would like to get more eyes on (please limit to 1 link only directly to the piece of content)...

**Comment by mikeysan** [comment]
I am learning [...] at the moment and struggling with...
